automate_git.py

```diff
@@ -4,23 +4,30 @@
 from configparser import ConfigParser
 from pathlib import Path
 
-# config file contains user settings
-config_file = 'config.ini'
-config = ConfigParser()
-config.read(config_file)
 
-# get all settings from config file
-access_token = config.get('your_settings', 'github_token')
-repo_name = config.get('your_settings', 'repo_name')
-first_file = config.get('your_settings', 'commit_file')
-msg = config.get('your_settings', 'first_commit_msg')
-github_name = config.get('your_settings', 'github_name')
+def setup_config():
```

Comment:
@funbeedev  - how about passing the config file in to the function as well...

```python
def setup_config(config_file = None):
    if config_file is None:
        config_file = 'config.ini'
```

Default to `None` is the standard way of [defaulting parameters](https://docs.python.org/3/tutorial/controlflow.html#default-argument-values).